style/swift/README.md

```diff
@@ -11,3 +11,4 @@ Swift
 * Let the compiler infer the type whenever possible
 * Group computed properties below stored properties
 * Use a blank line above and below computed properties
+* Group `private` methods in a `private` `extension`
```

Comment:
how about

> group methods with different levels of access control in specific extensions.

That's horribly worded, but the gist is: we should be doing this for `public` as well.